### DISCUSSION TITLE
New RyzenTosh

### DISCUSSION THREAD

**Discussion by jdzmfg** [sileshn/Ryzentosh#41] [General]
We have similar specs but not totally the same, same motherboard but my cpu is a ryzen 5 4500 and my gpu is 1050ti ssc, im looking to run high sierra or newer if possible with my setup, I tried a bootable usb earlier and got a "End SetConsoleMode" error. I would really appreciate some help.

**Comment by sileshn** [comment]
https://www.reddit.com/r/hackintosh/comments/15lmw9p/success_ventura_open_core_nvidia_1050ti/

https://www.youtube.com/watch?v=YwHXwbpIRBg

**Reply by jdzmfg** [reply]
https://www.reddit.com/r/hackintosh/s/ZxUvdduGtX

**Comment by sileshn** [comment]
Your config file has errors.

OpenCanopy.efi is loaded at UEFI->Drivers, but Misc->Boot->PickerMode is not set to External!
CheckMisc returns 1 error!

OpenVariableRuntimeDxe at UEFI->Drivers[24] must have LoadEarly set to TRUE!
OpenRuntime.efi at UEFI->Drivers[22] should have its LoadEarly set to TRUE when OpenVariableRuntimeDxe.efi at UEFI->Drivers[24] is in use!
OpenRuntime.efi (currently at UEFI->Drivers[22]) should be placed after OpenVariableRuntimeDxe.efi (currently at UEFI->Drivers[24])!
OpenVariableRuntimeDxe.efi is incompatible with FirmwareSettingsEntry.efi!
OpenUsbKbDxe.efi at UEFI->Drivers[23] should NEVER be used together with UEFI->Input->KeySupport!
OpenUsbKbDxe.efi at UEFI->Drivers[23], and Ps2KeyboardDxe.efi at UEFI->Drivers[25], should NEVER co-exist!
CheckUefi returns 6 errors!

**Comment by jdzmfg** [comment]
Those are now fixed, this is my updated log: https://gofile.io/d/XCkbR5

**Comment by jdzmfg** [comment]
I don't know which ones to disable or enable

OCS: No schema for InstanceIdentifier at 20 index, context <Quirks>! 
OCS: No schema for DisableIoMapperMapping at 22 index, context <Quirks>! 
OCS: No schema for ConsoleFont at 15 index, context <Output>! 
OCS: No schema for GopBurstMode at 16 index, context <Output>! 
OCS: No schema for InitialMode at 17 index, context <Output>! 
OCS: No schema for PciIo at 18 index, context <ProtocolOverrides>! 
OCS: No schema for ResizeUsePciRbIo at 14 index, context <Quirks>! 

Serialisation returns 7 errors!

**Comment by sileshn** [comment]
Send me your EFI folder

**Comment by jdzmfg** [comment]
so some how some way it booted to the installation menu, I formatted one of drives to install mac os 10.13 but it got stuck on the continue button (greyed out). everything on my drive is in this zip: https://gofile.io/d/hEw9Da

**Comment by sileshn** [comment]
Your EFI is too bloated with unwanted stuff. Its easier to start afresh and add data/kexts only required for your setup.
